test/testcafe/screenshot-test.js

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { statSync } from 'fs';
+
+fixture `Screenshot`
+    .page('https://google.com');
+
+test('Take screenshot', async t => {
+    var screenshotPath = await t.takeScreenshot();
```

Comment:
Let's keep it undocumented. And let's use custom path here (you can use `tmp` package to generate paths)